config/jail.conf

```diff
@@ -344,7 +344,7 @@ logpath = %(lighttpd_error_log)s
 [roundcube-auth]
 
 port     = http,https
-logpath  = /var/log/roundcube/userlogins
+logpath  = /var/log/roundcubemail/errors
```

Comment:
On Tue, 26 May 2015, Lee Clemens wrote:
>  > -logpath  = /var/log/roundcube/userlogins
>  > +logpath  = /var/log/roundcubemail/errors

>    I can explain this in more detail if needed, but my aim was to make
>    default fail2ban work with default roundcube - today it does not
>    (regardless of platform). There are a lot of How-To's out there for
>    getting fail2ban and roundcube to work, I just wanted to finally correct
>    fail2ban.

and thanks for that ;)

well -- default one for this one is quite vague beast since

./config/defaults.inc.php:$config['log_dir'] = RCUBE_INSTALL_PATH . 'logs/';


and INSTALL_PATH by default is

define('INSTALL_PATH', dirname($_SERVER['SCRIPT_FILENAME']).'/');

if I see it correctly.... 

But what I meant is that there possibly was a reason why it was pointing to
/var/log/roundcube/userlogins... good to have git and sensible commit messages,
so it stems from using an additional plugin
http://mattrude.com/projects/roundcube-fail2ban-plugin/ to log those failed
attempts.  I guess it is no longer necessary?  If not, then indeed we could
just migrate to a new default (which is imho again somewhat vague) and specify
per each OS that canonical location.


-- 
Yaroslav O. Halchenko, Ph.D.
http://neuro.debian.net http://www.pymvpa.org http://www.fail2ban.org
Research Scientist,            Psychological and Brain Sciences Dept.
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik